connector/routingconnector/router.go

```diff
@@ -120,7 +121,12 @@ func (r *router[C]) registerRouteConsumers() error {
 		if !ok {
 			route.statement = statement
 		} else {
-			r.logger.Warn("statement already exists in the routing table, the existing routing item will be overwritten")
+			pipelineNames := []string{}
+			for _, pipeline := range item.Pipelines {
+				pipelineNames = append(pipelineNames, pipeline.Name())
+			}
+			exporters := strings.Join(pipelineNames, ", ")
+			r.logger.Warn(fmt.Sprintf(`statement "%s" already exists in the routing table, the route with exporters %s will be ignored`, item.Statement, exporters))
```

Comment:
```suggestion
			r.logger.Warn(fmt.Sprintf(`statement %q already exists in the routing table, the route with exporters %q will be ignored`, item.Statement, exporters))
```

Reply:
The suggestion was based on an earlier commit, recommitted it. Thanks for the quick feedback. :)